data-collection-scripts/fda-covid-ivd-euas/scrape.py

```diff
@@ -0,0 +1,52 @@
+import requests
+import csv
+import sys
+
+from bs4 import BeautifulSoup
+
+def scrape():
+    url = "https://www.fda.gov/emergency-preparedness-and-response/mcm-legal-regulatory-and-policy-framework/emergency-use-authorization#covidinvitrodev"
+    page = requests.get(url)
+    soup = BeautifulSoup(page.text, 'html.parser')
+    rows = soup.find('table').find_all('tr')[1:]
```

Comment:
What would you think about throwing an exception if `soup.find()` is `None`, or some quick validation that the data is there, since something will have gone horribly wrong in that case and we'll want an email from GitHub telling us the action is broken?

Reply:
Sure